tox.ini

```diff
@@ -41,8 +42,26 @@ commands =
     black tests --check --diff
     flake8 tests
     mypy tests
+
+[testenv:tests]
+description = Run checks on the test cases in tests/
+
+extras =
+    {[testenv:tests-base]extras}
+
+commands =
+    {[testenv:tests-base]commands}
     pytest -n auto
 
+[testenv:tests-develop]
+description = Run checks on the test cases in tests/, including tests for development forks
+
+extras =
+    {[testenv:tests-base]extras}
+
+commands =
+    pytest -n auto --until={[main]development_fork}
+
```

Comment:
I wonder if we could optionally specify an evm binary to tox either by specifying it via:
1. A command-line flag.
2. Environment variable.
and then provide this value to `--evm-bin` in the pytest call?

That would make it even more useful.

We could do that for the non-development version, too.

Reply:
It seems like it's possible by doing this: https://tox.wiki/en/3.3.0/example/general.html

Do you mean that we should build the two binaries from the start and then call tox twice only changing `--evm-bin` ?

Reply:
> It seems like it's possible by doing this: https://tox.wiki/en/3.3.0/example/general.html

Nice, thanks for checking!

> Do you mean that we should build the two binaries from the start and then call tox twice only changing `--evm-bin` ?

This is one possibility, but as I mentioned in the follow-up comment, it's probably not worth adding this now. It'll add additional complexity that we won't need in the long run (as later I imagine we'll start specifying which evm-version to use locally in some test paths that target functionality for forks under development).

I really like the simplicity of this approach as it stands.